Spanner/tests/System/TransactionTest.php

```diff
@@ -30,17 +31,19 @@ class TransactionTest extends SpannerTestCase
     const TABLE_NAME = 'Transactions';
 
     private static $row = [];
-
+    
```

Comment:
Why?
```suggestion

```

Reply:
The new line character was there already. So thought of keeping it. 
Ref: https://github.com/googleapis/google-cloud-php/blob/main/Spanner/tests/System/TransactionTest.php#L32

Please let me know if you would like me to change this.